cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -41,6 +41,13 @@ spec:
     # port: 8443
     # serve the dashboard using SSL
     # ssl: true
+  # enable prometheus alerting for cluster
+  monitoring:
+    # requires Prometheus to be pre-installed
+    enabled: true
+    # namespace to deploy prometheusRule in. If empty, namespace of the cluster will be used.
+    # Recommended: Use same rulesNamespace for all clusters (if no specific requirement)
```

Comment:
For all Ceph clusters? So perhaps the namespace should be different than the default Ceph cluster namespace? According to your comment, it seems recommended to have the same namespace for all clusters so perhaps it should be different from one Ceph cluster?

Reply:
> For all Ceph clusters? 

Yes.
> So perhaps the namespace should be different than the default Ceph cluster namespace?

Not necessarily. It could be anything. The idea is to avoid duplicates across namespaces which will create dupicate alerts (prometheus doesn't provide duplication check).
 
> According to your comment, it seems recommended to have the same namespace for all clusters so perhaps it should be different from one Ceph cluster?

It is better to keep it different from ceph cluster namespaces. We could add it to the prometheus namespace, say "monitoring". But, this is totally up-to the admin as we can not add any restrictions.

Reply:
Could you add more details here about the `rulesNamespace`? For example:
- If you have a single rook-ceph cluster, set the `rulesNamespace` to the same namespace as the cluster
- If you have multiple rook-ceph clusters in the same k8s cluster, choose the same namespace to set `rulesNamespace` for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.

Reply:
ack